config/default-lsst.psfex

```diff
@@ -11,8 +11,8 @@ PSF_ACCURACY    0.01            # Accuracy to expect from PSF "pixel" values
 #------------------------- Point source measurements -------------------------
  
 CENTER_KEYS     slot_Centroid_x,slot_Centroid_y # Catalogue parameters for source pre-centering
-PHOTFLUX_KEY    slot_ApFlux_flux    # Catalogue parameter for photometric norm.
-PHOTFLUXERR_KEY slot_ApFlux_fluxErr # Catalogue parameter for photometric error
+PHOTFLUX_KEY    base_CircularApertureFlux_9_0_flux      # Catalogue parameter for photometric norm
+PHOTFLUXERR_KEY base_CircularApertureFlux_9_0_fluxSigma # Catalogue parameter for photometric error
```

Comment:
I presume that has always been the understanding.  The only difference here from what is being run on HSC is that we are using a 9 pixel radius for the `psfex` PHOTFLUX_KEY instead of 7 (because the list of aperture radii includes: 3, 4.5, 6, 9, 12, 17, 25, 35, 50, 70).  The aperture corrections on both are tied to 12 pixels.  So, if you think the difference between 7 and 9 in the 'psfex' setting is potentially dangerous, I guess we could add/alter the list of circular aperture radii measured.  Note, I'm pretty sure I ran a quick "before and after" on this one (where "before" was set to 6 pixels) and I didn't see any striking differences (although, admittedly, I did not save the results...if you'd like to see a comparison, I can certainly do it again and post some figures).